crd/nodeinfo/api/v1alpha/groupversion_info.go

```diff
@@ -0,0 +1,23 @@
+//go:build !ignore_uncovered
+// +build !ignore_uncovered
+
+// Package v1alpha contains API Schema definitions for the acn v1alpha API group
+// +kubebuilder:object:generate=true
+// +groupName=acn.azure.com
+package v1alpha
```

Comment:
you've copy/pasted the mistake made in the NNC. conventionally this should be `v1alpha1`

Reply:
updated